core/reachability.cpp

```diff
@@ -31,7 +31,15 @@ bool ReachabilityMap::initializeReachabilityMapping(const IR::Node *node,
         symbolMap[symbol.get()].emplace(node);
     }
 
-    return emplace(node, cond).second;
+    auto it = find(node);
```

Comment:
When an element exists already using emplace it will return the pointer to the element, then you can just append. Like
`map[key].push_back(value)` should work.